whitepaper/chapters/system.tex

```diff
@@ -26,7 +26,7 @@ \subsection{Transaction Plugins}
 	catapult::plugins::PluginManager& manager);
 \end{lstlisting}
 
-\class{PluginManager} provides access to the subset of configuration that plugins need to initialize themselves.
+\class{PluginManager} provides access to the subset of configurations that plugins need to initialize themselves.
```

Comment:
"subset of configurations" sounds strange consider rewording to something else 

e.g.
restricted view
portion of configuration
partial configuration

Reply:
Finally kept "subset of configuration", sounds more natural.